enterprise/admin/views.py

```diff
@@ -199,8 +200,19 @@ def _enroll_users(cls, emails, course_details, mode, request):
             ).format(enrolled_count=enrolled_count, course_id=course_id))
         if non_existing:
             messages.warning(request, _(
-                "The following users do not have accounts yet and were not enrolled: {}"
+                "The following users do not have accounts yet and were not yet enrolled: {}"
```

Comment:
I'm not sure why we're using the past tense in the second part of the sentence. Also, are both conditions true (they don't have accounts AND they're not enrolled), or does just one have to be true?

Assuming only one has to be true, how about this:

The following users do not have <platform name> accounts or are not enrolled in this course.

Assuming both have to be true, I'd suggest this:

The following users do not have <platform name> accounts and are not enrolled in this course.